RNTester/android/app/build.gradle

```diff
@@ -69,7 +69,8 @@ project.ext.react = [
     bundleAssetName: "RNTesterApp.android.bundle",
     entryFile: file("../../js/RNTesterApp.android.js"),
     root: "$rootDir",
-    inputExcludes: ["android/**", "./**", ".gradle/**"]
+    inputExcludes: ["android/**", "./**", ".gradle/**"],
+    extraPackagerArgs: ["--reactNativePath", "$rootDir"]
```

Comment:
should we update template too ?

Reply:
Nope, this only affects projects that run React Native from source. RNTester uses `react-native` from source (`../` folder). All other apps get `react-native` from `node_modules/react-native`